tests/test_other.py

```diff
@@ -6197,6 +6203,7 @@ def test_ld_library_path(self):
     self.assertContained('Hello2', out)
     self.assertContained('Hello3', out)
     self.assertContained('Hello4', out)
+    self.assertNotContained('Floating-point remainder failed', out)
```

Comment:
notContained is somewhat fragile - how about printing "Ok." in an else to the error check, and verifying the that success notification is present?